static/docs/commands-reference/remote_add.md

```diff
@@ -252,6 +252,46 @@ variables:
 
 </details>
 
+<details>
+
+### Click for Aliyun OSS
+
+First you need to setup OSS storage on Aliyun Cloud and then use s3 style url for oss storage and make endpoint a configurable value, an example is shown below:
+```dvc
+$ dvc remote add myremote oss://my-bucket/path
+```
+To set key id, key secret and endpoint you need to use modify command from dvc, a sample ussge is show below:
+```dvc
+$ dvc remote modify myremote oss_key_id my-key-id
+$ dvc remote modify myremote oss_key_secret my-key-secret
+$ dvc remote modify myremote oss_endpoint endpoint
+```
+You can also set enviornment variables and use them later, to set enviornment variables use following enviormnent variables:
+```dvc
+$ export OSS_ACCESS_KEY_ID="my-key-id"
+$ export OSS_ACCESS_KEY_SECRET="my-key-secret"
+$ export OSS_ENDPOINT="endpoint"
+```
+
+#### Test your oss storage using docker
+
+Start a container running an oss emulator.
```

Comment:
add an extra new line before and after code blocks for readability

Reply:
@shcheklein  done.